extensions/github/package.json

```diff
@@ -127,7 +137,14 @@
           "when": "github.hasGitHubRepo && resourceScheme != untitled && remoteName != 'codespaces'",
           "group": "0_vscode@0"
         }
-      ]
+      ],
+      "scm/historyItem/context": [
+				{
+					"command": "github.openCommitOfHistoryItem",
+					"when": "github.hasGitHubRepo",
```

Comment:
I will look into populating some context key to clearly indicate whether a history item is present on the remote(s) since the action should only be shown for history items that have been pushed to at least one remote.